package.json

```diff
@@ -68,7 +68,7 @@
     "webpack-dev-middleware": "^3.7.2",
     "webpack-log": "^2.0.0",
     "ws": "^6.2.1",
-    "yargs": "12.0.5"
+    "yargs": "15.3.1"
```

Comment:
We can't use `15` version right now, sorry

Reply:
Sorry haven't seen the other PR and issue ^^' 
thx